src/sonic-bgpcfgd/bgpcfgd/static_rt_timer.py

```diff
@@ -16,6 +16,9 @@ def __init__(self):
 
     def set_timer(self):
         """ Check for custom route expiry time in STATIC_ROUTE:EXPIRY_TIME """
+        keys = self.db.keys(self.db.APPL_DB, "STATIC_ROUTE_EXPIRY_TIME")
+        if len(keys) == 0:
```

Comment:
Should we add a log here to prove this thread is still alive? Maybe info/debug level?

Reply:
When key not exists. there will be log periodicly printed like :
 WARNING bgp#bgpcfgd: :- operator(): DB '{APPL_DB}' is empty with pattern 'STATIC_ROUTE_EXPIRY_TIME'!



When key exist there will be log periodicly printed  like:
INFO bgp#bgpcfgd: Static route expiry set to 160s